examples/post_training_quantization/torch/ssd300_vgg16/main.py

```diff
@@ -0,0 +1,191 @@
+"""
+ Copyright (c) 2023 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import os
+import re
+import subprocess
+from pathlib import Path
+from tqdm import tqdm
+
+import torch
+import nncf.torch
+
+import nncf
+import openvino.runtime as ov
+import torchvision
+from fastdownload import FastDownload
+from nncf.torch.dynamic_graph.context import patch_with_no_nncf_trace
```

Comment:
I am mostly ok with the changes but, since it is supposed to be a public API, it better to expose it on a higher level, e.g.
`from nncf.torch import ...`. 
I would also rename it to `disable_tracing(...)`.
Finally, it should be `from nncf.torch import disable_tracing`.